imports/ui/reducers/CardReducer.js

```diff
@@ -11,6 +12,7 @@ const initialState = {
       item: "Bananas",
       price: "$0.99",
       uuid: "41048778-017e-44d6-9496-6b60a7c81d8f",
+      count: 10,
```

Comment:
Please pull in master's changes and update the add item caller to add an initial count to the `item` payload.

Reply:
Can we also rename to "rating" instead of count?